linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_groupownership/oval/shared.xml

```diff
@@ -0,0 +1,52 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("All User Files and Directories In The Home Directory Must Be Group-Owned By The Primary User") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_files_groupownership"
+                 comment="All User Files and Directories In The Home Directory Must Be Group-Owned By The Primary User"/>
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_files_groupownership_objects" version="1">
+    <unix:username datatype="string" operation="not equal">nobody</unix:username>
+    <filter action="include">state_accounts_users_home_files_groupownership_interactive_gids</filter>
+  </unix:password_object>
+
+  <unix:password_state id="state_accounts_users_home_files_groupownership_interactive_gids" version="1">
+    <unix:user_id datatype="int" operation="greater than or equal">{{{ gid_min }}}</unix:user_id>
```

Comment:
Should the `root` not considered an interactive user?

Reply:
https://github.com/ComplianceAsCode/content/pull/7790#discussion_r744794969